public/app/features/plugins/datasource_srv.ts

```diff
@@ -327,6 +327,12 @@ export class DatasourceSrv implements DataSourceService {
       }
     }
 
+    if (!filters.pluginId) {
+      sorted = sorted.filter((x) => {
+        return matchPluginId('prometheus', x.meta);
+      });
+    }
+
```

Comment:
Maybe we should add some tests for this added logic?

Reply:
removed this as it's not needed